akka-persistence/src/main/scala/akka/persistence/PersistentActor.scala

```diff
@@ -21,6 +21,14 @@ case object RecoveryCompleted extends RecoveryCompleted {
   def getInstance = this
 }
 
+/** Convenience for skipping recovery of [[PersistentActor]] instances. */
+object NoRecovery {
```

Comment:
ah, I'd rather `extends Recovery(toSequenceNr = 0L)`, but extending case classes is not that nice
what about removing this and adding a factory method in `Recovery` companion. E.g. `Recovery.noRecovery`?